task_schedule3.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env /proj/sot/ska/bin/perl
+#!/usr/bin/env perlska
```

Comment:
If the Ska environment (and PATH) are not already defined at this point, then `#!/usr/bin/env perlska` is undefined (i.e. no guarantee it will be found).

Reply:
OK.  Yeah, that's a good point.  So the best thing is to have a full path to a conda installed perlska when installed.  This is fine from a conda perspective, but what should live as the shebang in the git repo?

Reply:
Alternatively, I think we could define PATH in the crontab, but that's less self-documenting.

Reply:
`acis_taco` is the template here.  Cron tab has
```
/proj/sot/ska3/flight/bin/skare task_schedule3.pl ...`
```
So `/proj/sot/ska3/flight/bin/skare` is universal way to get into Ska3 and to set PATH so `$SKA/bin/task_schedule3.pl` is found.  Any perl or Python share task scripts should use `/usr/bin/env perl` or `/usr/bin/env python`.